Prow: update secrets population (#4045)
kyma-project/test-infra: docs/prow/obligatory-security-measures.md

## Change Secrets regularly

> All Secret used in the Prow production cluster must be changed every six months. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, [External Secrets Syncer](../../prow/workload-cluster/README.md) will change all Secrets in the Prow cluster automatically.

Reviewer: ```suggestion
All Secrets used in the Prow production cluster must be changed every six months. Follow the [Prow secret management](./prow-secrets-management.md) guidelines to create a new key ring and new Secrets. Once the key ring and new Secrets are ready, the [External Secrets Syncer](../../prow/workload-cluster/README.md) changes all Secrets in the Prow cluster automatically.
```
The link ->  [External Secrets Syncer](../../prow/workload-cluster/README.md) doesn't point the External Secrets Syncer

### Change Secrets

> Change all Secrets that were valid when the person was a project member. Follow [Prow secret management](./prow-secrets-management.md) to create a new key ring and new Secrets. Then, [External Secrets Syncer](../../prow/workload-cluster/README.md) will change all Secrets in the Prow cluster automatically.

Reviewer: ```suggestion
Change all Secrets that were valid when the person was a project member. Follow the [Prow secret management](./prow-secrets-management.md) guidelines to create a new key ring and new Secrets. Once the key ring and new Secrets are ready, the [External Secrets Syncer](../../prow/workload-cluster/README.md) changes all Secrets in the Prow cluster automatically.
```
The link ->  [External Secrets Syncer](../../prow/workload-cluster/README.md) doesn't point the External Secrets Syncer
